### PR TITLE
Mta 1178 application archetypes

### PR DESCRIPTION
**Pull request by HagayVider1**
Replaces https://github.com/windup/windup-documentation/pull/760

### CODE REVIEW

File: docs/topics/mta-web-archetypes.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+[id="mta-web-application-archetypes_{context}"]
+= Application archetypes
+
+{ProductName}({ProductShortName}) enables assessing and analyzing entire groups of applications or *archetypes*, according to common charateristics.
```
Comment (emarcusRH) [inline docs/topics/mta-web-archetypes.adoc:9]: Typo: "charateristics. " >>>  characteristics

File: docs/topics/mta-web-archetypes.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+[id="mta-web-application-archetypes_{context}"]
+= Application archetypes
+
+{ProductName}({ProductShortName}) enables assessing and analyzing entire groups of applications or *archetypes*, according to common charateristics. 
+
+Application *archetypes* are dfined according to criteria tags and application taxonomy. Each archetype selects how the applications are assessed according to its characterstics.
```
Comment (emarcusRH) [inline docs/topics/mta-web-archetypes.adoc:11]: Typo: "dfined" >>> defined

File: docs/topics/mta-web-archetypes.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+:_content-type: PROCEDURE
+
+[id="creating-archetype_{context}"]
+== Creating an application Archetype
+
+You can create an Archetype that defines the charateristics of a group of applications, which you can assess as a group.
+
+.Procedure
+
+. In *Migration* view, click *Aarchetypes*.
```
Comment (emarcusRH) [inline docs/topics/mta-web-archetypes.adoc:22]: Typo: *Aarchetypes* >>> *Archetypes*

File: docs/topics/mta-web-archetypes.adoc (hunk 1 above)
Comment (anarnold97) [inline docs/topics/mta-web-archetypes.adoc:9]:
```suggestion
{ProductName}({ProductShortName}) enables assessing and analyzing entire groups of applications or *archetypes*, according to common characteristics.
```

File: docs/topics/mta-web-archetypes.adoc [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+[id="mta-web-application-archetypes_{context}"]
+= Application archetypes
+
+{ProductName}({ProductShortName}) enables assessing and analyzing entire groups of applications or *archetypes*, according to common characteristics.
+
+Application *archetypes* are dfined according to criteria tags and application taxonomy. Each archetype selects how the applications are assessed according to its characterstics.
```
Comment (anarnold97) [inline docs/topics/mta-web-archetypes.adoc:11]:
```suggestion
Application *archetypes* are defined according to criteria tags and application taxonomy. Each archetype selects how the applications are assessed according to its characterstics.
```

File: docs/topics/mta-web-assessment-module.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+== Opening an assessment questionnaire
+
+The {ProductName} ({ProductShortName}) assesses applications that are to be migrated according to a set of questions relevant to the application, such as dependencies, *[more question types]*. {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.
+
+.To open a saved questionnaire
+
+// Get updated image for MTA
+image::images/assessment-process-start-01.png[]
+
+. Open the {ProductName} user interface and click *Application Inventory* in the left menu bar. *Application assessment* appears in the main pane, listing all the available questionnaires.
```
Comment (sshveta) [inline docs/topics/mta-web-assessment-module.adoc:37]: *Application assessment* DOES NOT appears in the main pane, listing all the available questionnaires.

We need to click on application's kebab menu to see the "assess" button .

Screenshot from 2024-01-26 09-03-08

**Pull request closed without merging**